.github/workflows/eager-package-main.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
         arch: [grayskull, wormhole_b0]
     runs-on: ${{ matrix.os }}
     environment: production
+    env:
+      LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
```

Comment:
Noooo not this one pls, it doesn't use builders and we need to be testing no `LD_LIBRARY_PATH` in production

Reply:
removed